.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           key:
             ubuntu-build-${{ env.cache-name }}-${{
             hashFiles('plugin/test/fixtures/**/package.json') }}-node-modules
-      - run: npm install -g netlify-cli
+      - run: npm install -g netlify-cli@18.0.0
```

Comment:
there seems to be regression in latest version of cli (18.0.1) that result in skipping rewrites to `.netlify/functions/__api` and hitting framework's dev server directly which trips up the tests - I'm pinning it for now to "fix" the tests

I suspect it's caused by https://github.com/netlify/cli/pull/6994 and reported in https://netlify.slack.com/archives/C07686YAY13/p1737134106866159?thread_ts=1736521386.323879&cid=C07686YAY13